eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -38,7 +38,7 @@ parameters:
   default: []
 - name: TestTimeoutInMinutes
   type: number
-  default: 60
+  default: 90
```

Comment:
Given we are overriding this in core\ci.yml already I don't think we need to update this default value.